.github/workflows/ci.yaml

```diff
@@ -31,7 +31,6 @@ jobs:
     secrets: inherit
 
   run-k8s-tests-on-aks:
-    if: ${{ contains(github.event.pull_request.labels.*.name, 'ok-to-test') }}
```

Comment:
@fidencio are we going to leave without the 'ok-to-test' or you plan to move to somewhere else?

Reply:
Those were moved here: https://github.com/kata-containers/kata-containers/blob/e067d183336ce0bb4f79d922787d44bc1e6144f4/.github/workflows/ci-on-push.yaml#L21

So, in order to start the CI you still have to set `ok-to-test`

Reply:
oh, my old eyes didn't see it. thanks!